cmd/generator/main.go

```diff
@@ -395,12 +338,135 @@ func getReleases(ctx context.Context, client *github.Client, repoName string, in
 		if resp.NextPage == 0 {
 			break
 		}
-		opts.Page = resp.NextPage
+		options.Page = resp.NextPage
 	}
 
 	return result, nil
 }
 
+func getReleasePlugin(release *github.RepositoryRelease, repository *github.Repository, existingPlugins []*model.Plugin) (*model.Plugin, error) {
+	var releaseName string
+	if release.GetName() == "" {
+		releaseName = release.GetTagName()
+	} else {
+		releaseName = fmt.Sprintf("%s (%s)", release.GetName(), release.GetTagName())
+	}
+	logger.Debugf("found latest release %s", releaseName)
+
+	downloadURL := ""
+	signatureAssets := make([]github.ReleaseAsset, 0)
+	releaseNotesURL := release.GetHTMLURL()
+	var updatedAt time.Time
+	for _, releaseAsset := range release.Assets {
+		assetName := releaseAsset.GetName()
+		if strings.Contains(assetName, "-amd64") {
+			// Ignore old style bundles
```

Comment:
nit: worth a debug message in case we later try to build a database of all plugin versions?